build.gradle

```diff
@@ -102,6 +100,7 @@ dependencies {
     //Unmanaged Dependencies
     testImplementation group: 'com.jayway.jsonpath', name: 'json-path', version: '2.2.0'
     testImplementation group: 'com.jayway.jsonpath', name: 'json-path-assert', version: '2.2.0'
+    testImplementation group: 'com.google.guava', name: 'guava', version: '30.1-jre'
```

Comment:
Curious what this line is about

Reply:
Previously this was a transitive dependency from the swagger library. Removing swagger also removed this JAR which broke the tests since they make use of it.